src/vs/editor/standalone/browser/accessibilityHelp/accessibilityHelp.ts

```diff
@@ -333,7 +333,8 @@ class ShowAccessibilityHelpAction extends EditorAction {
 				linux: {
 					primary: KeyMod.Alt | KeyMod.Shift | KeyCode.F1,
 					secondary: [KeyMod.Alt | KeyCode.F1]
-				}
+				},
+				kbExpr: EditorContextKeys.focus
```

Comment:
Could this be the best action and it gets overridden by higher priority ones? You might already have this in mind for a follow up

Reply:
Yes follow up